tests/crypto/secure-zkp-cli.test.js

```
@@ -0,0 +1,378 @@
+  let circuitFilesExist = true;
+
+  // Check if circuit files exist
+  beforeAll(() => {
+    circuitFilesExist = fs.existsSync(wasmPath) && fs.existsSync(zkeyPath) && fs.existsSync(vkeyPath);
+    if (!circuitFilesExist) {
+      console.warn(`
+        Warning: Circuit files not found. Skipping secure ZKP tests.
+        Please run 'npm run zkp:setup' to set up the ZKP authentication system.
+      `);
+    }
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeAll duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)